include/pika_slot.h

```diff
@@ -8,7 +8,7 @@
 #include "strings.h"
 
 const std::string SlotKeyPrefix = "_internal:slotkey:4migrate:";
-const size_t MaxKeySendSize = 10 * 1024;
+const size_t MaxKeySendSize = 1024;
```

Comment:
won't be necessary after https://github.com/Qihoo360/pink/commit/51e4bbeed2c9dbf2f728400369fc9d19d482bfe2#diff-9b6af85329e3015c44cf81fb59fd1686
take effects in pika?